dali/operators/generic/reduce/reduce.cc

```diff
@@ -29,6 +29,10 @@ DALI_SCHEMA(ReduceBase)
 
 Not providing any axis results in reduction of all elements.)code",
     std::vector<int>{ })
+  .AddOptionalArg("axes_names", R"code(Axis or axes along which reduction is performed, given by name.
```

Comment:
```suggestion
  .AddOptionalArg("axis_names", R"code(Axis or axes along which reduction is performed, given by name.
```
That's how we call it in other ops

Reply:
Done